test/Run.tests.amber

```diff
@@ -0,0 +1,28 @@
+#!/bin/sh
+#TEST-PROGRAM sander
+#TEST-DESCRIP TO_BE_DEtermined
+#TEST-PURPOSE regression, basic
+#TEST-STATE   undocumented
+
+. ../../../../test/program_error.sh
+
+if [ -z "$TESTsander" ]; then
+   TESTsander="${AMBERHOME}/bin/quick"
+fi
+
+if [ -z "$DO_PARALLEL" ]; then
+   export DO_PARALLEL=" "
+fi
+
+if [ -z "$QUICK_BASIS" ]; then
+  export QUICK_BASIS="../basis"
+fi
+
+for i in {ene,opt}*.in
+do
+  $DO_PARALLEL $TESTsander $i < /dev/null || error
```

Comment:
How does this work? Is there an error function defined?

The tests that I wrote for Amber are derived from sander tests that use csh, which has completely different syntax but provides a goto function:
```
#!/bin/csh -f
#TEST-PROGRAM sander
...
$DO_PARALLEL $TESTsander -O -c $inpcrd -o $mdout < dummy || goto error
...
/bin/rm -f restrt mdin mdinfo dummy
exit(0)

error:
echo "  ${0}:  Program error"
exit(1)
```
This then prints the name of the test script and Program error, and returns 1 if the program fails for some reason.

Reply:
`error` is defined in the following line:

> . ../../../../test/program_error.sh

This script will only work when executed from $AMBERHOME/AmberTools/src/quick/test

Reply:
This makes sense. Thanks for explaining!